### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/blog/category-and-tags.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#580] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/blog/category-and-tags.html

你可以通过 frontmatter 来为文章配置分类与标签，使它们出现在特定的分类与标签页面中。

https://theme-hope.vuejs.press/zh/guide/blog/category-and-tags.html

**Comment by wangdongovo** [comment]
测试

**Reply by esyka114514** [reply]
# 测试
